### PR TITLE
/network/utils/tunneldigger: Add Tunneldigger package

### PR DESCRIPTION
**Pull request by robimarko**
Add Tunneldigger package.

Tunneldigger is one of the projects of wlan slovenija open wireless network. It is a simple VPN tunneling solution based on L2TPv3 tunnels supported in recent Linux kernels.
It is used for VPN tunneling in wlan slovenija and otvorenamreza.org wireless networks.

Building tested in lede-sdk-17.01.1-ar71xx-generic_gcc-5.4.0_musl-1.1.16.Linux-x86_64.
Builds without issues.

Signed-off-by: Robert Marko 
robimarko@gmail.com

### CODE REVIEW

**Comment by NeoRaider** [comment]
Thanks for your contribution!

The signed-off-by line should have the same format as generated by `git commit -s` (`Signed-off-by: Foo bar <foo@bar.net>`), as scripts are sometimes used to parse these lines.

Also, I think the repo https://github.com/openwrt/packages (which is used by both OpenWrt and LEDE) is more appropriate for this package, there is no reason why it needs to be part of the LEDE base repository. I'm closing this here, it would be great if you could open a new PR against the packages repo.

Further comments inline.

File: package/network/utils/tunneldigger/Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tunneldigger
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+PKG_REV:=235e111fb8fa02c4687af7f695e21204d9d28fe6
```
Comment (NeoRaider) [inline package/network/utils/tunneldigger/Makefile]: Please also add a PKG_MAINTAINER definition.

File: package/network/utils/tunneldigger/files/tunneldigger.init [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+			sleep 1
+		done
+		echo "  tunneldigger stopped"
+	done
+}
+
+restart() {
+	stop
+	start
+}
```
Comment (NeoRaider) [inline package/network/utils/tunneldigger/files/tunneldigger.init]: The *restart* function can be dropped, the default one does the same.

**Comment by robimarko** [comment]
Thank you.
I will open a PR against linked repo.
Thanks for the tip about signed-off line

**Pull request closed without merging**